addons/project/static/src/components/project_view_task_button/project_view_task_button.js

```diff
@@ -0,0 +1,30 @@
+/** @odoo-module **/
+
+import { registry } from "@web/core/registry";
+import { useService } from "@web/core/utils/hooks";
+import { Component } from "@odoo/owl";
+
+export class ProjectViewTaskButton extends Component {
+    static template = 'project.ProjectViewTaskButton';
+    setup() {
+        this.action = useService("action");
+    }
+
+    async onViewTask () {
+        const record = this.props.record;
+        await record.save({ no_reload: true });
+        return this.action.doAction({
+            type: "ir.actions.act_window",
+            res_model: record.resModel,
+            res_id: record.data.id,
+            views: [[false, "form"]],
+            context: record.context,
+        });
```

Comment:
Shouldn't this be an orm call? :slightly_smiling_face:

Reply:
Hello @auon-odoo 

Yes, it should be an orm call.
Changes are applied. Thank you :slightly_smiling_face: